### DISCUSSION TITLE
个人打包的Linux语雀，中文输入时出现了重复

### DISCUSSION THREAD

**Discussion by WHIJK** [tw93/Pake#157] [Q&A]
因为系统原因，所以重新编译了一个语雀，但是出现GIF中中文输入问题【英文输入没有问题】，不清楚是不是就我一个遇到，求解惑啊~~~
系统：深度20.8【底层是debian】
![录屏_选择区域_20221212184303](https://user-images.githubusercontent.com/62982639/207026048-e30796ad-d479-4ad0-b38b-fa0083015ba4.gif)

**Comment by tw93** [comment]
试试本身的网页版本是不是有这个问题，换上说换一个输入法试试，看着像本身语雀的bug

**Reply by WHIJK** [reply]
是语雀的Bug了，打包的其他编辑器是没有问题的，感谢解答~~

**Reply by Tlntin** [reply]
ubuntu一样。

**Reply by Tlntin** [reply]
貌似不是语雀的bug。
新dev分支（用了新底层框架）打包的语雀是正常的。
不过我用的ubuntu，所以打包给你也用不了。晚点我更新一下dev分支的pake-cli命令行打包工具。

**Comment by WHIJK** [comment]
> 貌似不是语雀的bug。 新dev分支（用了新底层框架）打包的语雀是正常的。 不过我用的ubuntu，所以打包给你也用不了。晚点我更新一下dev分支的pake-cli命令行打包工具。

感谢啦，不过上周cli更新的时候，重新打包了一个试试，发现已经正常了
